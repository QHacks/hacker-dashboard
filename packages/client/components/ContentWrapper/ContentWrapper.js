import React from "react";

export default (props) => (
  <div
    css={`
      max-width: 1400px;
      margin-left: auto;
      margin-right: auto;
      padding-left: 145px;
      padding-right: 145px;
      height: 100%;
      @media screen and (max-width: 1200px) and (min-width: 860px) {
        padding-left: 80px;
        padding-right: 80px;
      }
      @media screen and (max-width: 860px) {
        padding-left: 5vw;
        padding-right: 5vw;
      }
    `}
  >
    {props.children}
  </div>
);
