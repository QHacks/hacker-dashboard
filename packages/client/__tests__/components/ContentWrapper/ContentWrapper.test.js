import React from "react";
import ContentWrapper from "../../../components/ContentWrapper/ContentWrapper";
import { shallow } from "enzyme";

describe("ContentWrapper", () => {
  it("renders correctly", () => {
    expect(shallow(<ContentWrapper />)).toMatchSnapshot();
  });
});
