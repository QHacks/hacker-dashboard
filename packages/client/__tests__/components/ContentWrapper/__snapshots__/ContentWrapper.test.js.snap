// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ContentWrapper renders correctly 1`] = `
<div
  className="css-118rwy6"
/>
`;
